compiler/src/dotty/tools/dotc/core/SymDenotations.scala

```diff
@@ -600,7 +600,7 @@ object SymDenotations {
 
     /** Is this a denotation of a stable term (or an arbitrary type)? */
     final def isStable(implicit ctx: Context) =
-      isType || is(Stable) || !(is(UnstableValue) || info.isInstanceOf[ExprType])
+      isType || !is(Ghost) && (is(Stable) || !(is(UnstableValue) || info.isInstanceOf[ExprType]))
```

Comment:
We should avoid marking the symbols as stable if they are ghost. We could assert that a symbol must at most one of those flags.

Reply:
Because in some places we might ask if `sym.is(Stable)` which would say true for a symbol with Stable and Ghost flags.

Reply:
Actually don't do this change